docs/datatype/image/image2image.md

```diff
@@ -1,60 +1,53 @@
 
 ## Build the Flow
 
-The solution to the problem entails a simple pipeline that can be subdivided into two steps:  **Index** and **Query**
+The solution uses a simple pipeline that can be subdivided into two steps:  **Index** and **Query**
 
 ### Index
 
-To search something out of the full data, first we need to index the data. What it means is that we store the embeddings
-of all the images from the pets dataset in some form of storage. The images can be read as a numpy array which is then
-fed to neural network of our choice. This neural network encodes the input images into some latent space which we call
-as embeddings. We then use **Indexer** to store these embeddings in memory.
+To search something out of the full dataset, we first need to index the data. This means that we store the embeddings
+of all the images from the dataset in some form of storage. The images can be read as a numpy array which is then
+fed to the neural network of our choice. This neural network encodes the input images into some latent space which we call
+"embeddings". We then use an **Indexer** to store these embeddings in memory.
 
 ### Query
 
-Once the data is indexed, i.e. our database is built, we simply need to feed our query which is an image or set of
-images to the model to encode it into embeddings and then use the **Indexer** to retrieve matching images. The matching
-can be based on any type of metrics but without going deeper into this, we will focus only on euclidean distance between
-two embeddings (corresponding to two images) as metrics.
+Once the data is indexed, i.e. our database is built, we simply need to feed our query (an image or set of
+images) to the model to encode it into embeddings and then use the **Indexer** to retrieve matching images. The matching
+can be based on any type of metric but without going deeper into this, we will focus only on Euclidean distance between
+two embeddings (corresponding to two images).
 
-Now, one might think what this *Indexer* is, or how to use neural network of our choice. Worry not, we've got you
-covered. In Jina AI, we have three fundamental concepts which is all you need to know to follow this tutorial. If you
-haven't read it yet, head on to [Jina's docs](https://docs.jina.ai/) page and give it a shot. Executor is the
-algorithmic unit in the Jina. It performs a single task on a `Document` or `DocumentArray`.
-
-We have many executors available at [Jina Hub](https://hub.jina.ai) - a marketplace for Executors. You can use any of
-them relevant to your tasks or build one of your own. Coming back to problem, we will use **SimpleIndexer** executor as
-our indexer (the one that stores and retrieves data). This executor also returns us the matching `Document` when we make
+We will use the **SimpleIndexer** Executor as
+our indexer (the one that stores and retrieves data). This Executor also returns the matching `Document` when we make
```

Comment:
```suggestion
The **Indexer** executor should support vector search along with indexing. For this tutorial we will create our own **Indexer** based on `SimpleIndexer` executor available on Jina Hub. This Executor returns the matching `Document` when we make
```

Reply:
**Automated message**

## Capitalization

Please capitalize first letters of the following (when used to reference project or component):

- Flow
- Executor
- Document
- Jina
- Jina Hub
- Python
- Docker
- Kubernetes

### Note!

This only applies **outside** a code or CLI context. If you're describing Python, YAML, any programming language, or command line input/output, please use the capitalization you would use **in that context**.

##### Right

```python
from jina.flow import Flow
```

##### Wrong

```python
from Jina.Flow import Flow
```

---

Made with [GitHub saved replies](https://github.com/settings/replies)

Reply:
Remove "should"